src/app/containers/CpsText/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,178 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CpsTextContainer with data should render correctly 1`] = `
+.c1 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  color: #3F3F42;
+  padding-bottom: 1.5rem;
+  margin: 0;
+}
+
+.c0 {
+  grid-column: 1 / span 6;
+}
+
+.c2 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  list-style-type: none;
+}
+
+.c2 > li::before {
+  content: ' ';
+  display: inline-block;
+  width: 2rem;
+  background: url("data:image/svg+xml,%3Csvg height='10' width='10' xmlns='http://www.w3.org/2000/svg'%3E%3Ccircle cx='5' cy='5' r='3'%3E%3C/circle%3E%3C/svg%3E") no-repeat center;
```

Comment:
Have we checked that this can be seen ok in windows high contrast mode and when the user changes the colour preferences in firefox?

Reply:
This has been fixed in https://github.com/bbc/psammead/pull/2698